docs/docs/programmatically-create-pages-from-data.md

```diff
@@ -21,37 +21,29 @@ access to the
 [`createPage`](/docs/actions/#createPage) action
 which is at the core of programmatically creating a page.
 
-```javascript:title=gatsby-node.js
-exports.createPages = ({ graphql, actions }) => {
-  const { createPage } = actions
-  return new Promise((resolve, reject) => {
-    graphql(`
-      {
-        allMarkdownRemark {
-          edges {
-            node {
-              fields {
-                slug
-              }
-            }
-          }
-        }
-      }
-    `).then(result => {
-      // highlight-start
-      result.data.allMarkdownRemark.edges.forEach(({ node }) => {
-        createPage({
-          path: node.fields.slug,
-          component: path.resolve(`./src/templates/blog-post.js`),
-          context: {
-            slug: node.fields.slug,
-          },
-        })
-      })
-      // highlight-end
-      resolve()
+```js:title=gatsby-node.js
+exports.createPages = async function({ actions, graphql }) {
+  // highlight-start
```

Comment:
This should be moved down 👇 (where it was previously!)

Reply:
@DSchau Now? :D Sorry this is my first docs fix. Still figuring it out.

Reply:
Please don't apologize!! We appreciate you taking the time--and this looks great now.

Thank you!